kernel/indTyping.ml

```diff
@@ -65,31 +65,35 @@ let mind_check_names mie =
 (************************************************************************)
 
 type univ_info = { ind_squashed : bool; ind_has_relevant_arg : bool;
-                   ind_min_univ : Universe.t option; (* Some for template *)
-                   ind_univ : Universe.t;
-                   missing : Universe.Set.t; (* missing u <= ind_univ constraints *)
+                   ind_min_univ : Sorts.t option; (* Some for template *)
+                   ind_univ : Sorts.t;
+                   missing : Sorts.t list; (* missing u <= ind_univ constraints *)
                  }
 
+let sup_sort s1 s2 =
+  let open Sorts in
+  sort_of_univ (Universe.sup (univ_of_sort s1) (univ_of_sort s2))
```

Comment:
should be defined in Sorts

Reply:
No, because it doesn't make sense, it would be a partial function but here we know statically this cannot fail. Next step of the changes is to reimplement template poly without `Universe.t` anyways.